### ISSUE TITLE
HyperparameterChecker - Explore which hyperparameters should be explicitly checked in pytorch and tensorflow and implement them

### ISSUE THREAD

**Issue by Hynn01** [Hynn01/dslinter#13]
Explore which hyperparameters should be explicitly checked in pytorch and tensorflow.

**Comment by Hynn01** [comment]
Learning rate, batch size, momentum, and weight decay are important parameters that need to be set.
Smith, Leslie N. "A disciplined approach to neural network hyper-parameters: Part 1--learning rate, batch size, momentum, and weight decay." arXiv preprint arXiv:1803.09820 (2018).
